angr/simos/windows.py

```diff
@@ -171,7 +173,8 @@ def state_entry(self, args=None, env=None, argc=None, **kwargs):
 
         return state
 
-    def state_blank(self, **kwargs):
+    def state_blank(self, fs=None, concrete_fs=False, chroot=None, cwd=b'/Users/User', pathsep=b'/',
```

Comment:
Isn't pathsep `\\` on Windows?

Reply:
I've made some tests and open('/Users/Saullo/doc.txt', 'rb') opens the file 'C:\Users\Saullo\doc.txt'. I've made the mistake of assume that it is a common behavior. As I am not sure about this, I am changing it to common Windows path notation.

Reply:
`SimFile` works perfectly for Linux, but https://github.com/angr/angr/blob/b9de8302be0ac1ac0e1beece7013a3cd73ce3eb2/angr/state_plugins/filesystem.py#L126-L127 and https://github.com/angr/angr/blob/b9de8302be0ac1ac0e1beece7013a3cd73ce3eb2/angr/state_plugins/filesystem.py#L148 assumes that `pathsep` always represents filesystem root path, what is not true for Windows filesystems.

On Windows filesystems, `b'\\'` is the root of the current drive. Is this good enough?